skglm/datafits/group.py

```diff
@@ -71,3 +71,78 @@ def gradient_scalar(self, X, y, w, Xw, j):
 
     def intercept_update_step(self, y, Xw):
         return np.mean(Xw - y)
+
+
+class LogisticGroup(BaseDatafit):
```

Comment:
Could it inherit from Logistic to avoid duplicating code ?

Reply:
Agree!

**Side note**: There is a small pitfall in this case as we are using jitclasses: we can't call the constructor of the ``Logistic`` (I get a numba compilation error because of the ``super`` keyword).